testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/health/MicroProfileHealthSecuredHTTPEndpointEmptyMgmtUsersTestCase.java

```diff
@@ -31,11 +32,11 @@ public class MicroProfileHealthSecuredHTTPEndpointEmptyMgmtUsersTestCase {
     @ContainerResource
     ManagementClient managementClient;
 
-    @Deployment
+    @Deployment(testable = false)
     public static Archive<?> deployment() {
-        final Archive<?> deployment = ShrinkWrap.create(JavaArchive.class, "MicroProfileHealthSecuredHTTPEndpointEmptyMgmtUsersTestCase.jar")
-                .addClasses(MicroProfileHealthSecuredHTTPEndpointSetupTask.class, EmptyMgmtUsersSetupTask.class);
-        return deployment;
+        // Dummy deployment to trigger @ServerSetup
+        return ShrinkWrap.create(JavaArchive.class, "MicroProfileHealthSecuredHTTPEndpointEmptyMgmtUsersTestCase.jar")
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
```

Comment:
FYI re AssumeTestGroupUtil.emptyJar/War/Ear.

Those methods are in that class because they got added in the context of needing to deal with the fact that Arq would deploy things even though we would later disable the test when a @BeforeClass was called. So checking conditions and returning an empty artifact was a workaround.

That's not a problem any more because now WFARQ let's us stop everything via an AssumptionViolatedException from a ServerSetup.